components/ProfileMenu.vue

```diff
@@ -192,21 +193,34 @@ const saveAndCloseContentLanguageDialog = () => {
                 :class="{
                   'bg-label-separator -separator': active,
                 }"
-                class="flex w-full rounded-lg px-3 py-2"
+                class="flex w-full rounded-lg px-3 py-2 font-medium text-[15px]"
                 href="mailto:bmm-support@bcc.no"
                 target="_blank"
               >
                 {{ $t("profile.contact") }}
               </a>
             </MenuItem>
+            <MenuItem v-slot="{ active }" as="div">
```

Comment:
Won't the version be shown two times on Web this way..?

Reply:
Yes. Maybe it's better to deviate from Søren's design and add the logo on the desktop app.

Reply:
re-added the logo and removed the profile entry